R/pkg/inst/worker/daemon.R

```diff
@@ -18,10 +18,11 @@
 # Worker daemon
 
 rLibDir <- Sys.getenv("SPARKR_RLIBDIR")
-script <- paste(rLibDir, "SparkR/worker/worker.R", sep = "/")
+dirs <- strsplit(rLibDir, ",")[[1]]
+script <- paste(dirs[[1]], "SparkR/worker/worker.R", sep = "/")
```

Comment:
also, shouldn't you use the `File.separator` equivalent as the `sep`? What if this is being run on Windows?